.github/workflows/verify.yml

```diff
@@ -101,15 +105,14 @@ jobs:
     container: ghcr.io/xnvme/xnvme-deps-citools-latest:next
 
     steps:
-    - name: Retrieve the source-archive
+    - name: Retrieve, the xNVMe source archive
```

Comment:
The added comma here doesn't make a lot of sense. Same goes for the rest of the cases